include/experimental/__p2630_bits/submdspan_extents.hpp

```diff
@@ -24,6 +24,59 @@
 namespace MDSPAN_IMPL_STANDARD_NAMESPACE {
 namespace detail {
 
+MDSPAN_TEMPLATE_REQUIRES(
+  class Unsigned,
+  /* requires */ (std::is_unsigned_v<Unsigned>)
+)
+MDSPAN_INLINE_FUNCTION
+constexpr auto abs(const Unsigned &val) {
+  return val;
+}
+
+MDSPAN_TEMPLATE_REQUIRES(
+  class Unsigned, Unsigned val,
+  /* requires */ (std::is_unsigned_v<Unsigned>)
+)
+MDSPAN_INLINE_FUNCTION
+constexpr auto abs(const std::integral_constant<Unsigned, val>&) {
```

Comment:
In general, this design would need to work for any _`integral-constant-like`_ type, not just `std::integral_constant`.  Also, [P3663R3](https://isocpp.org/files/papers/P3663R3.html) just passed wording review today and may change C++26, depending on the results of the C++ Standard Committee plenary meeting next month.  That may affect this design.